src/main/java/net/imagej/legacy/ImageJ2Options.java

```diff
@@ -80,10 +80,20 @@
 		description = "<html>Whether to use ImageJ2's file I/O mechanism when "
 			+ "opening files.<br>Image files will be opened using the SCIFIO library "
 			+ "(SCientific Image<br>Format Input and Output), which provides truly "
-			+ "extensible support for<br>reading and writing image file formats.",
+			+ "extensible support for<br>reading and writing image file formats.</html>",
```

Comment:
Well, that HTML is the bastardized Java version, which Swing uses for rich text formatting. AFAIK, you absolutely do not need the closing tag. And in any case: adding it here is unrelated to the stated purpose of your commit.